experimental/CHANGELOG.md

```diff
@@ -57,6 +57,7 @@ All notable changes to experimental packages in this project will be documented
 
 * Add `resourceDetectors` option to `NodeSDK` [#3210](https://github.com/open-telemetry/opentelemetry-js/issues/3210)
 * feat: add Logs API @mkuba [#3117](https://github.com/open-telemetry/opentelemetry-js/pull/3117)
+* feat(instrumentation): implement `require-in-the-middle` singleton [#3161](https://github.com/open-telemetry/opentelemetry-js/pull/3161) @mhassan1
```

Comment:
I think this should go under the "unreleased" enhancement section since this PR wasn't part of 0.33.0 release.